runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/StreamingEngineClientTest.java

```diff
@@ -480,6 +366,10 @@ private StreamingEngineConnectionState waitForWorkerMetadataToBeConsumed(
     return connections.get();
   }
 
+  private void waitForFirstWorkerMetadata() {
+    while (!Preconditions.checkNotNull(streamingEngineClient).isWorkerMetadataReady()) {}
```

Comment:
This should have some sleep so it's not busy spinning cpu now that it just polls without waiting.

But see above, perhaps this can just be removed?

Reply:
Removed